### DISCUSSION TITLE
Any way to add empty bars?

### DISCUSSION THREAD

**Discussion by AlejandroGDiaz** [CoderLine/alphaTab#1256] [General]
First of all, very amazing library! I have been using it for the past months and I really love what it has been done in it!

I would like to ask something though, Is there a proper or optimal way to add empty bars to a tab? Maybe by modifiying something in the data model?

If there is a way, maybe you can share an example about how it could be done, please.

Thanks :)

**Comment by Danielku15** [comment]
There is nothing out-of-the-box from my side but you might check out how things are done in https://github.com/CoderLine/alphaTab/discussions/1141 

I have it on the roadmap to support easier score editing APIs, currently modifying the score model is a bit of a sensitive thing as things have to be consistent in the model. 

Can you share more details about what exactly you wanna build? I assume adding empty bars serves some purpose.

**Reply by AlejandroGDiaz** [reply]
Hi, thanks for the fast response!

I am thinking about adding empty bars for the type of cases where: I display multiple versions of the same song (horizontally) and since some tabs tend to have more bars than others, then I would align them accordingly by adding empty bars and therefore they would become easier to compare (bar-to-bar).

It will be awesome when you get add score editing API's! I will still check into the solution you suggested.
